jest/mock-expo-sqlite.js

```diff
@@ -85,13 +86,28 @@ class SQLiteDatabase {
       return;
     }
 
-    callback(null, results);
+    try {
+      callback(null, results);
+    } catch (e) {
+      if (!this._allowUnhandled) {
+        throw e;
+      }
+    }
   }
 
   close() {
     this._closed = true;
     this._db.close();
   }
+
+  /**
+   * Suppress some otherwise-unhandled Promise rejections.
```

Comment:
I think being vague here is basically OK, because only a few tests will use it.

Probably a try/catch around that other `callback` call site would make sense. I'll add that. That might also give me a clean less-vague description to give this `allowUnhandled` method.